lib/thrift/protocol/binary.ex

```diff
@@ -105,7 +106,7 @@ defmodule Thrift.Protocol.Binary do
   end
   def serialize(:application_exception, %TApplicationException{message: message, type: type}) do
     <<@string::size(8), 1::16-signed, byte_size(message)::size(32), message::binary,
-      @i32::size(8), 2::16-signed, type::32-signed, 0::size(8)>>
+      @i32::size(8), 2::16-signed, type::32-signed, @stop::8-signed>>
```

Comment:
I like `@stop` as a hint to the reader. Why make it signed though? It's reasonable to expect the reader to understand that elements in a binary expression are unsigned bytes unless otherwise annotated. You see it in `iex` all the time, for instance. So I'd just end with `@stop>>` here. Probably drop the `::size(8)` where we use it, too.